circle.yml

```diff
@@ -0,0 +1,135 @@
+machine:
+  timezone:
+    Europe/Vienna
+  environment:
+    ZEMU_: "appemu23"
+    EMU_: "appemu21"
+    XEMU_: "appemu10"
+    Zsdpath: "/sdcard"
+    sdpath: "/storage/sdcard"
+    Xsdpath: "/mnt/sdcard"
+    ANDROID_HOME: /usr/local/android-sdk-linux
+    ANDROID_SDK: /usr/local/android-sdk-linux/
+    _SDK_: /usr/local/android-sdk-linux/
+    GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx1000m -XX:+HeapDumpOnOutOfMemoryError"'
+    GRADLE_HOME: ~/g/gradle-2.10
+    PATH: ~/g/gradle-2.10/bin:$PATH
+general:
+  artifacts:
+    #-./app/build/outputs/logs/
```

Comment:
Huh, why don't you want to change it? The usual way is to either give hints why something should stay in it's current way or comply with the reviewer and change it. Btw., changing is easy, just push a new commit to the branch.